src/app/store/slices/plan/index.ts

```diff
@@ -15,15 +15,19 @@ export interface PlanState {
   isLoadingPlanLimit: boolean;
   isLoadingPlanUsage: boolean;
   individualPlan: StoragePlan | null;
+  businessPlan: StoragePlan | null;
   teamPlan: StoragePlan | null;
   planLimit: number;
   planUsage: number;
   usageDetails: UsageResponse | null;
-  subscription: UserSubscription | null;
+  subscription: UserSubscription | null; //TODO: Please review this field, it may be necessary to remove it and use the field for each subscription separately
```

Comment:
Have you identified any reason why we might prefer to have it separate? Looking through this I can't think of any